.github/workflows/docs_test.yml

```diff
@@ -0,0 +1,54 @@
+name: Docs_test
+
+on: [push, pull_request]
+
+jobs:
+  CI:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.9]
+
+    steps:
+    - name: apt-get stuff needed for libstell and vmec
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y build-essential gfortran openmpi-bin libopenmpi-dev libnetcdf-dev libnetcdff-dev liblapack-dev libscalapack-mpi-dev libhdf5-dev libhdf5-serial-dev git m4 libfftw3-dev libboost-all-dev libopenblas-dev
```

Comment:
Since vmec and spec are not installed I think we can drop `gfortran libscalapack-mpi-dev m4 libfftw3-dev`. If we don't install booz_xform and virtual_casing then I think we can also drop `libnetcdf-dev libnetcdff-dev liblapack-dev  libhdf5-dev libhdf5-serial-dev libopenblas-dev`. Since MPI isn't used, perhaps we can also drop `openmpi-bin libopenmpi-dev`